.travis.yml

```diff
@@ -54,29 +41,17 @@ jobs:
               - sh -e /etc/init.d/xvfb start
               - sleep 3
           script:
-              - if [ "$WEB_PROJECTS" = "true" ]; then travis_wait 30 npm run test:e2e -- --since $TRAVIS_BRANCH; else echo "No individuals web package changed. Skipping tests"; fi
-        # - name: "iOS Automated tests"
```

Comment:
Now the detox tests are only commented, please keep in mind we need to re-enable it soon

Reply:
We always can resurrect this code from git history. But probably we will have to re-write it differently anyway.